charter.md

```diff
@@ -0,0 +1,41 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+The team is responsible for maintaining [Compat 2021](https://wpt.fyi/compat2021), [Interop 2022](https://wpt.fyi/interop-2022), and defining future interoperability efforts and metrics.
+
+## Scope
+
+The team works by prioritizing specific web platform features. The minimum requirements for a feature to be in scope are:
+
+- The feature is defined by a specification from the [IETF](https://www.ietf.org/), [Khronos Group](https://www.khronos.org/), [TC39](https://tc39.es/), [W3C](https://www.w3.org/), or [WHATWG](https://whatwg.org/). Other specifications can be considered on a case-by-case basis.
+- The feature can be tested in [web-platform-tests](https://github.com/web-platform-tests/wpt). Other test suites can be considered on a case-by-case basis.
+
+Browser features that don't affect web developers, like bookmarks, are out of scope.
+
+Writing specifications is not in scope, but the team can liaise with and work within other working groups to make progress in key areas.
+
+## Governance
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition.
+
+Decisions can be made both in the repository and in meetings, as long as consensus can be established. Meeting time and agenda should be announced at least 48 hours in advance, and meeting minutes posted in the repository.
+
+The Interop effort is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+Substantive changes to this charter are done via the [web-platform-tests RFC process](https://github.com/web-platform-tests/rfcs).
+
+## Team Members
+
+The following organizations make up the Interop Team, in alphabetical order:
+
+* Apple
+* Bocoup
+* Google
+* Igalia
+* Microsoft
+* Mozilla
+
+## Joining the Team
+
+If your organization has a track record of contributions to the web platform you can request to join the Interop Team. Broad participation is welcome, but the existing team will make a decision based on consensus. There are also [other ways of contributing](./CONTRIBUTING.md).
```

Comment:
```suggestion
If your organization has a track record of contributions to the implementation of web platform features, you may request to join the Interop Team. The decision will be made by the current [Team Members](#team-members) based on consensus.

You do not have be part of the interop Team to participate. There are also [other ways of contributing](./CONTRIBUTING.md).
```

How does an organization request to join? Open an issue? A Template? -> #124

Reply:
Good question!

I think we should also define that an organization can be removed by their own request, or by consensus to deal with the case that they're no longer around to request leaving.

Reply:
I've updated this bit now. I think that "implementation of web platform features" was an improvement to what I had written originally, but I changed it to "contributions to the web-platform-tests project" to unambiguously fit all current members, and also it makes sense given the team is part of that project.